### PR TITLE
virsh.attach_detach_disk: PV cleanup

### PR DESCRIPTION
**Pull request by will-Do**
Signed-off-by: Yanbing Du <ydu@redhat.com>

### CODE REVIEW

**Comment by will-Do** [comment]
And another patch https://github.com/avocado-framework/avocado-vt/pull/495 is needed to fix the vgcreate failure in this case

File: libvirt/tests/src/virsh_cmd/domain/virsh_attach_detach_disk.py [hunk 1]
```diff
@@ -418,6 +418,7 @@ def acpiphp_module_modprobe(vm, os_type):
             if test_logcial_dev:
                 libvirt.delete_local_disk("lvm", vgname=vg_name, lvname=lv_name)
                 lv_utils.vg_remove(vg_name)
+                utils.system("pvremove %s" % device_source, ignore_status=True)
```
Comment (waynesun09) [inline libvirt/tests/src/virsh_cmd/domain/virsh_attach_detach_disk.py]: is pvremove needed after each vg_remove? if so, maybe we could add it into vg_remove with an pv clean option to enable or disable it

Comment (will-Do) [inline libvirt/tests/src/virsh_cmd/domain/virsh_attach_detach_disk.py]: Basically, we don't need do this, after iscsi target logout, the PV will be removed automatically. But, yes, we can consider to add this step(which may fail so ignore_status=True)

Comment (waynesun09) [inline libvirt/tests/src/virsh_cmd/domain/virsh_attach_detach_disk.py]: Ok, ack and merge this now, thx

**Pull request merged**